ptx/sec_deriv_inverse_function.ptx

```diff
@@ -317,7 +333,12 @@
         <caption>A right triangle defined by <m>y=\sin^{-1}(x/1)</m> with the length of the third leg found using the Pythagorean Theorem</caption>
           <!-- START figures/fig_inverse3.tex -->
         <image xml:id="img_inverse3" width="47%">
-          <description></description>
+        <shortdescription>right angle triangle defined by y=sin inverse (x/1)</shortdescription>
```

Comment:
you could write 'arcsin' instead of 'sin inverse'. Or perhaps even easier: "The right triangle defined by the equation sin(y)=x/1"